newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -523,6 +530,19 @@ const Instruction = (props: Props) => {
                 tabIndex={0}
                 id={id}
               >
+                {showDeprecatedInstructionWarning && metadata.isHidden() ? (
+                  <IconButton
+                    key="deprecated"
+                    size="small"
+                    tooltip={
+                      props.isCondition
+                        ? t`Deprecated condition`
+                        : t`Deprecated action`
+                    }
+                  >
+                    <Warning style={{ color: warningColor }} />
+                  </IconButton>
```

Comment:
I think you can avoid using an IconButton (this isn't really a button as you can't interact)
I'd go for wrapping the <Warning /> with a `<Tooltip></Tooltip>` from material-ui directly (check other parts of the code)

I would also suggest using `fontSize="small"` for the warning component, so that it stays quite small and doesn't increase the height of the line?